presidio-analyzer/presidio_analyzer/analyzer_utils.py

```diff
@@ -73,3 +73,213 @@ def is_verhoeff_number(input_number: int):
         for i in range(len(inverted_number)):
             c = __d__[c][__p__[i % 8][inverted_number[i]]]
         return __inv__[c] == 0
+
+    # fmt: off
+    in_vehicle_foreign_mission_codes = [
```

Comment:
Are these generic or referring just to the in_vehicle_registration? If it's just for this, can we please move it to the recognizer file?

Reply:
these are indeed specific to Vehicle Registration only and I don't see any future use elsewhere. Moved the logic back to recognizer classfile as advised.